akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -354,6 +355,16 @@ private final class Sdk(
     }
   }
 
+  private def isNotExcluded(clz: Class[_]): Boolean = {
```

Comment:
could we avoid double negation here?
`isExcluded` and then 

```
componentClasses
    .filter(hasComponentId)
    .filterNot(isExcluded)
```

Reply:
done

Reply:
please, `filterNot(i => !i.isNotExcluded)` 😄